lib/commons/color/get-background-color.js

```diff
@@ -65,9 +77,48 @@ function calculateObscuringAlpha(elmIndex, elmStack) {
 }
 
 function elmOutsideParent(elm, bgElm, bgColor) {
-	var visible = (elm !== bgElm && !dom.visuallyContains(elm, bgElm) && bgColor.alpha !== 0);
-	return visible;
+	var outside = (elm !== bgElm && !dom.visuallyContains(elm, bgElm) && bgColor.alpha !== 0);
+	if (outside) {
+		axe.commons.color.incompleteData.set('bgColor', {
+			node: elm,
+			reason: 'elmOutsideParent'
+		});
+	}
+	return outside;
 }
+
+// document.elementsFromPoint misses some elements we need
+// i.e. TR is missing from table elementStack and leaves out bgColor
+// https://github.com/dequelabs/axe-core/issues/273
+function includeMissingElements(elmStack, elm) {
```

Comment:
After a bit of wrangling this into a real test I'm going to opt for `<label>` instead of `<tr>`, as most browsers will "fix" a table when there is no TD. To keep it aligned with the rest of the changes in this PR I'll use an inline label so it won't get transformed by the browser.